components/job-orchestration/job_orchestration/retention/search_results_handler.py

```
@@ -0,0 +1,66 @@
+                continue
+
+            collection = results_cache_db.get_collection(collection_name)
+            collection_timestamp = _get_latest_doc_timestamp(collection)
+
+            if collection_timestamp < expiry_epoch:
+                logger.debug(f"Removing search results: {collection_name}")
+                _remove_result_metadata(results_cache_db, collection_name)
+                collection.drop()
+
+
+async def search_results_retention(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)